apps/settings/lib/Command/AdminDelegation/Add.php

```diff
@@ -50,7 +51,12 @@ public function execute(InputInterface $input, OutputInterface $output): int {
 			return 3;
 		}
 
-		$this->authorizedGroupService->create($groupId, $settingClass);
+		try {
+			$this->authorizedGroupService->create($groupId, $settingClass);
+		} catch (ConflictException) {
+			$io->warning('The ' . $settingClass . ' is already delegated to ' . $groupId . '.');
```

Comment:
```suggestion
			$io->warning('Administration of ' . $settingClass . ' is already delegated to ' . $groupId . '.');
```
For consistency with success.